tensorboard/components/vz_example_viewer/BUILD

```diff
@@ -0,0 +1,52 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tensorboard/defs:web.bzl", "tf_web_library")
+load("//tensorboard/defs:vulcanize.bzl", "tensorboard_html_binary")
+
+licenses(["notice"])  # Apache 2.0
+
+tf_web_library(
+    name = "vz_example_viewer",
+    srcs = [
+        "vz-example-viewer.html",
+        "vz-example-viewer.ts",
+    ],
+    path = "/vz-example-viewer",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//tensorboard/components/tf_imports:d3",
+        "//tensorboard/components/tf_imports:polymer",
+        "@org_polymer_iron_icons",
+        "@org_polymer_paper_button",
+        "@org_polymer_paper_card",
+        "@org_polymer_paper_checkbox",
+        "@org_polymer_paper_dialog",
+        "@org_polymer_paper_icon_button",
+        "@org_polymer_paper_input",
+        "@org_polymer_paper_radio_button",
+        "@org_polymer_paper_radio_group",
+        "@org_polymer_paper_slider",
+        "@org_tensorflow//tensorflow/core:example_protos_closure",
+    ],
+)
+
+tf_web_library(
```

Comment:
I'd be *very interested* in knowing if `bazel run` on this rule works, after you use `goog.require`. If so, please add a comment to this rule, and the binary beneath it, saying `# bazel run //etc/etc:demo`

Reply:
will look at in future PR